analyzers/its/Directory.Build.targets

```diff
@@ -5,6 +5,8 @@
     <PropertyGroup>
       <BinariesFolder>$(MSBuildStartupDirectory)\..\packaging\binaries</BinariesFolder>
       <AnalysisOutputPath>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)</AnalysisOutputPath>
+      <DiagnosticsPath>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(PROJECT)-$(AssemblyName)</DiagnosticsPath>
```

Comment:
Why is there a `PROJECT` in the file name, when it's already in the path as a parent directory?